package.json

```diff
@@ -36,7 +36,7 @@
     "husky": "^8.0.0",
     "prettier": "^2.8.8",
     "standard-version": "^9.5.0",
-    "turbo": "^1.10.7"
+    "turbo": "^1.9.8"
```

Comment:
Why is turbo version changed?

Reply:
because it caused errors with linting (something with GO binary..) so I downgraded it.